
Please check the Gevent compatible option in settings

File >> Settings >>Python Debugger >> Gevent Compatible



![Solution to debug exceptionin in pycharm](\Solution to debug exceptionin in pycharm.png)